code/AdobeEdge.brs

```diff
@@ -575,6 +561,10 @@ function _adb_task_node_EventProcessor(internalConstants as object, task as obje
         processQueuedRequests: function() as void
             if m.edgeRequestWorker.isReadyToProcess() then
                 responses = m.edgeRequestWorker.processRequests()
+                if responses = invalid
```

Comment:
check for roArray here only so we reduce the nesting below.

`if responses = invalid or Type(responses) <> "roArray" `